front/src/modules/companies/components/AddPersonToCompany.tsx

```diff
@@ -51,9 +55,23 @@ export const AddPersonToCompany = ({
   const [updatePerson] = useUpdateOnePersonMutation();
   const [insertOnePerson] = useInsertOnePersonMutation();
   const { refs, floatingStyles } = useFloating({
+    open: isDropdownOpen,
     placement: 'right-start',
-    middleware: [flip(), offset({ mainAxis: -20, crossAxis: 25 })],
+    middleware: [flip(), shift(), offset({ mainAxis: 30, crossAxis: 0 })],
   });
+  // close inline creation dialog
+  useListenClickOutside({
+    refs: [refs.floating],
+    callback: () => {
+      if (isCreationDropdownOpen) setIsCreationDropdownOpen(false);
+      if (isDropdownOpen) setIsDropdownOpen(false); // just for double check, it is handled in the SingleEntitySelect
```

Comment:
I think we should actually leave it to the picker / double text input to notify the listenClickOutside.
Can you try using: DoubleTextFieldInput instead? there is alredy a onOutsideClick Prop that you could use to close the dropdown

Reply:
Thank you for referring me at the component that fit here as better. But I think its DoubleTextInput not DoubleTextFieldInput. Because DoubleTextFieldInput is a specific meta field for user-generated ui. :)